### PR TITLE
imprecv:1.0.1

### PR DESCRIPTION
**Pull request by jskherman**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: Explain what the package does and why it's useful.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/imprecv/1.0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+## Usage
+
+This `imprecv` is intended to be used by importing the template's [package entrypoint](cv.typ) from a "content" file (see [`template.typ`](template/template.typ) as an example).
+In this content file, call the functions which apply document styles, show CV components, and load CV data from a YAML file (see [`template.yml`](template/template.yml) as an example).
+Inside the content file you can modify several style variables and even override existing function implementations to your own needs and preferences.
+
+### With the [Typst CLI](https://github.com/typst/typst)
+
+The recommended usage with the Typst CLI is by running the command `typst init @preview/imprecv:1.0.0` in your project directory.
```
Comment (elegaanz) [inline packages/preview/imprecv/1.0.1/README.md:85]: I think it should be `1.0.1` here.

**Review by elegaanz — COMMENTED** [review]
Looks good, just a version number that wasn't updated in the README.

File: packages/preview/imprecv/1.0.1/README.md (hunk 1 above)
Comment (jskherman) [reply packages/preview/imprecv/1.0.1/README.md]: Thanks for looking out, I have now updated the version.

**Comment by elegaanz** [comment]
Thank you!

**Pull request merged**